settings/template.ini

```diff
@@ -72,6 +72,13 @@ Methods[]=ftps
 Methods[]=webdav
 Methods[]=webdavs
 
+# Add support for text which is started from www. without protocol.
+# For example change text "www.ez.no" to link tags.
+LinkWwwUrlsWithoutProtocol=disabled
+
+HideProtocols[]
```

Comment:
This variable name is not so good, as it suggests by it's name (to us at any level) that the protocol text is hidden, which is not exactly true. Instead in truth, when this feature is used the protocols are striped from the generated link text. For these reasons we think this setting name should be changed. `StripLinkProtocols` is in our opinion a better name. We request you change this setting name.